_posts/2024-04-27-welcome.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Welcome to Women Coding Community
+date: 2024-04-27
+author: WCC Team
+blurb_img: /assets/images/blog/welcome.jpg
+description: |
+    Be a part of the change. Join us to help bridge the gender gap in technology. Together, we can create a future where women are leading drivers of technology and innovation.
+---
+
+<div>
+<h4>Hello and welcome to WCC!</h4>
```

Comment:
Accessibility-wise, these h4 should be replaced with h2.

Reply:
Fyi, I created this issue for creating resuable font size variables to match the defined typography styles defined in Figma: https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/38. So I think it's fine to wait with adjusting the headings to match designs, once we get those reusable variables.

Reply:
Yeah, that's alright✅ It was pretty much a sample blogpost to show how articles could look like.